ods_ci/tests/Tests/500__ide/502__ide_elyra.robot

```diff
@@ -107,10 +107,10 @@ Verify Pipelines Integration With Elyra Running Hello World Pipeline Test     #
     # We need to navigate to the page because the project name hold a state
     # In a fresh cluster, if not state found, it will select the first one
     # In this case, the first could not be the project created
-    Menu.Navigate To Page    Data Science Pipelines    Runs
+    Menu.Navigate To Page    Data Science Pipelines
     Select Pipeline Project By Name    ${PRJ_TITLE}
     Log    ${pipeline_run_name}
-    Verify Pipeline Run Is Completed    ${pipeline_run_name}    timeout=5m
+    Verify Pipeline Run Is Completed    ${pipeline_run_name}    timeout=5m    experiment_name="hello-generic-world"
```

Comment:
where does this come from? my experiments created from elyra were named "standard data science pipeline" when I tried this

Reply:
You are right. I've updated it